Simplenote/SimplenotePDFExporter.swift

```diff
@@ -0,0 +1,39 @@
+import Foundation
+
+class SimplenotePDFExporter {
+    func exportNoteToFiles(note: Note, filename: String) -> URL? {
+        let data = createPDFDataFromNote(note: note)
+        return writeNoteToDocuments(data: data, filename: filename)
+    }
+    
+    private func writeNoteToDocuments(data: Data, filename: String) -> URL? {
+        let targetURL = FileManager.documentsURL.appendingPathComponent(filename)
+        do {
+            try data.write(to: targetURL)
+        } catch {
+            NSLog("Note Exporter Failure: \(error)")
+            return nil
+        }
+        return targetURL
+    }
+    
+    private func createPDFDataFromNote(note: Note) -> Data {
+        let format = UIGraphicsPDFRendererFormat()
+        
+        let pageWidth = 8.5 * 72.0
+        let pageHeight = 11 * 72.0
```

Comment:
Can we move these to constants?

```
private struct Metrics {
   static let pageSize = CGSize(...)
}
```

Plus: what's the page size of these constants? (please add a comment, or name the variable afterwards)